cat-theme/src/css/base/_buttons.scss

```diff
@@ -0,0 +1,47 @@
+.btn {
+    &--main {
+
+        @include flex ($justify-content : flex-start);
+
+        border: 0.5px solid $color-header-text-light;
+        border-radius: 4rem;
+        height: 5rem;
+        font-size: 1.5rem;
+        transition: all 0.5s ease-in-out;
+        width: 17rem;
+
+        &:hover {
+            transform: scale(1.1) perspective(1px);
+        }
+
+        a {
+
+            @include flex ($justify-content : flex-start);
```

Comment:
Mixin's should be placed alphabetically as if `@include` was nonexistent.   You would alphabetize this as if it were `flex`.

Reply:
Why is there a space before and after the `:`?